ledger-service/http-json-cli/src/main/scala/com/daml/http/Config.scala

```diff
@@ -106,7 +106,12 @@ private[http] final case class JdbcConfig(
 )
 
 private[http] object JdbcConfig extends ConfigCompanion[JdbcConfig]("JdbcConfig") {
-  import dbbackend.ContractDao.supportedJdbcDriverNames
+
+  // FIXME Duplicated to not introduce a circular dependency between ContractDao and this
```

Comment:
Should I create a separate target for this that both `ContractDao` and this can depend on? Otherwise I'm not 100% of how to fix this.

Reply:
I don’t quite understand the issue. What is stopping us from either moving this to `ContractDao` and making the CLI depend on that or the other way around? That said, splitting things into separate targets is always a reasonable option imho.

Reply:
Because `ContractDao` is in `http-json` and if we make `http-json-cli` depend on `http-json` the dependency becomes circular. Or am I missing something?

Reply:
What I was trying to ask is, why can’t ContractDao use the definition here given that it already depends on it?

Reply:
Because `ContractDao` defines which are the supported drivers, which I think makes a lot of sense. I don't think we should put this in the CLI.

Reply:
Makes sense, then either a separate target just for this or a separate target for all of `ContractDao` to break the circular dependency seems sensible.

Reply:
This is going to be quite a large change due to some dependencies between various packages that are requiring me to break more and more packages apart. I would recommend we accept a one line code duplication for the time being while I break apart the modules into a separate PR.

Reply:
Tracked in https://github.com/digital-asset/daml/issues/9229

Reply:
perfectly fine